### PR TITLE
I made a new service, please check

### PR DESCRIPTION
**Pull request by ghost**
I made a new service, please check

### CODE REVIEW

File: online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/controller/BookstoreController.java [hunk 1]
```diff
@@ -1,24 +1,40 @@
...
+        log.info("Findig book by title", title);
+        final var book = service.findByTitle(title);
+        if (book.isEmpty()) {
+            return ResponseEntity.notFound().build();
+        } else {
+            return ResponseEntity.ok(book);
+        }
+    }
+
+    @PostMapping("/adbook")
```
Comment (KJudit01) [inline online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/controller/BookstoreController.java]:
```suggestion
    @PostMapping("/addbook")
```

Comment (KJudit01) [reply online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/controller/BookstoreController.java]: Be careful about the right spelling :)

File: online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/service/Service.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import com.groupA.onlinebookstoremanagementsystem.repository.BookstoreRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.List;
+
+@org.springframework.stereotype.Service
+@Slf4j
+@RequiredArgsConstructor
+public class Service {
```
Comment (KJudit01) [inline online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/service/Service.java]:
```suggestion
public class BookstoreService {
```

File: online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/service/Service.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        return book;
+    }
+
+    public Book addBook(Book book) {
+        return bookstoreRepository.save(book);
+    }
+    public void deleteBook(Long id){
+        bookstoreRepository.deleteById(id);
+    }
+}
```
Comment (KJudit01) [inline online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/service/Service.java:27]: Great solution! Well Done! :)

File: online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/service/Service.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+@org.springframework.stereotype.Service
+@Slf4j
+@RequiredArgsConstructor
+public class Service {
+    private final BookstoreRepository bookstoreRepository;
+
+    public List<Book> findByTitle(String title) {
+        final var book = bookstoreRepository.findByTitle(title);
+        return book;
```
Comment (KJudit01) [inline online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/service/Service.java:18]: Valami

File: online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/service/Service.java (hunk 3 above)
Comment (hollszabolcs) [reply online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/service/Service.java:27]: Thank you :)

File: online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/service/Service.java (hunk 4 above)
Comment (hollszabolcs) [reply online-bookstore-management-system/online-bookstore-management-system/src/main/java/com/groupA/onlinebookstoremanagementsystem/service/Service.java:18]: I'm so sorry

**Review by KJudit01 — APPROVED** [review]

**Pull request merged**